### PR TITLE
[feature] Add Bambu Cool Plate SuperTack

### PR DESCRIPTION
**Pull request by f0x52**
# Description
Cherry-picked commits from bambu-studio to add support for the new supertack plate. Depends on https://github.com/SoftFever/OrcaSlicer/pull/7629 to get the filament config support as well.
Fixes https://github.com/SoftFever/OrcaSlicer/issues/7664

## Tests
Built on NixOS Linux, did a little bit of usage testing and seems to work as expected

### CODE REVIEW

File: src/slic3r/GUI/Tab.cpp [hunk 1]
```diff
@@ -3622,9 +3627,17 @@ void TabFilament::toggle_options()
         toggle_line("adaptive_pressure_advance_model", has_adaptive_pa && pa);
         toggle_line("adaptive_pressure_advance_bridges", has_adaptive_pa && pa);
 
+<<<<<<< HEAD
```
Comment (TheDevMinerTV) [inline src/slic3r/GUI/Tab.cpp:3630]: There's a left-over merge conflict here

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: src/slic3r/GUI/Tab.cpp (hunk 1 above)
Comment (f0x52) [reply src/slic3r/GUI/Tab.cpp]: Oops, forgot to force-push after I fixed that, should be good now

**Review by Noisyfox — APPROVED** [review]
LGTM

**Comment by igiannakas** [comment]
PR raised here: https://github.com/SoftFever/OrcaSlicer/pull/7883

**Pull request merged**